pkg/descheduler/apis/config/types_pluginargs.go

```diff
@@ -145,3 +149,19 @@ type MigrationControllerArgs struct {
 	// DefaultDeleteOptions defines options when deleting migrated pods and preempted pods through the method specified by EvictionPolicy
 	DefaultDeleteOptions *metav1.DeleteOptions
 }
+
+type MigrationFrozenObjectType string
+
+const (
+	MigrationFrozenWorkload MigrationFrozenObjectType = "workload"
+)
+
+type FrozenGateMap map[MigrationFrozenObjectType]MigrationFrozenGate
+
+// MigrationFrozenGate means that if the specified dimension has multiple migrations within the configured time period
+// and exceeds the configured threshold, it will be frozen for a certain period of time.
+type MigrationFrozenGate struct {
```

Comment:
how about define as
``` go
type MigrationFrozenGate struct {
  Duration       metav1.Duration
  MaxMigrating   int
}
```
which means max migrate M pods in D duration (just like QPS=M/D, Burst=0)
what do you think @FillZpp @hormes @jasonliu747

Reply:
This configuration is indeed OK. The existing implementation is relatively simple, and a workload can be automatically processed through the cache mechanism, without an additional GC mechanism. If the ratelimit method is used, it means maintaining a ratelimit instance for each workload and specifying an additional cycle time to execute GC logic.

Reply:
updated with rate limiter.